spec/fixtures/config.toml

```diff
@@ -0,0 +1,12 @@
+port = 8443
+insecure_port = 8080
+database_url = "postgres://draupnir:draupnir@127.0.0.1/draupnir?sslmode=disable"
+data_path = "/draupnir"
+environment = "test"
+shared_secret = "thesharedsecret"
+oauth_redirect_url = "https://server.com/oauth_callback"
+oauth_client_id = "the_client_id"
+oauth_client_secret = "the_client_secret"
+tls_certificate_path = "/etc/ssl/certs/draupnir_cert.pem"
+tls_private_key_path = "/etc/ssl/certs/draupnir_key.pem"
+trusted_user_email_domain = "@gocardless.com"
```

Comment:
TOML lets you nest stuff under namespaces, so we could think about restructuring this a bit
```toml
[http]
port = 8443
insecure_port = 8080
tls_certificate_path = "..."
tls_private_key_path = "..."

[oauth]
redirect_url = "..."
client_id = "..."
client_secret = "..."
```
or whatever.

Reply:
Yeah this might be quite nice, cleans up the config file a bit. Happy with addressing that later though.

Reply:
If we address it later we have to do a second breaking change - better to do it now I think.